init/common.js

```diff
@@ -202,6 +206,19 @@ const createProject = async (interestId, userId, project, addTeam=true) => {
   });
 };
 
+const createOpportunity = async (interestId, opportunity) => {
+    let existing = await strapi.entityService.findMany("api::opportunity.opportunity", {filters: {title: opportunity.title}});
+    existing = existing[0];
+    if (existing) {
+      return;
+    }
+  
+    opportunity.interests = [interestId];
+    
+    const service = strapi.service("api::opportunity.opportunity");
+    existing = await service.create({ data: opportunity });
+};
```

Comment:
As we are not using existing after line 219, we can replace this:
```existing = await service.create({ data: opportunity });```
to just this ?
```await service.create({ data: opportunity });```